### PR TITLE
[enviroplushat.py] Add PM-sensor data, enable with variable

### PR DESCRIPTION
**Pull request by shaunmulligan**
Change-type: minor
Signed-off-by: Shaun Mulligan shaun@balena.io

### CODE REVIEW

File: sensor/scripts/enviroplushat.py [hunk 1]
```diff
@@ -36,17 +44,46 @@ def get_readings(self, sensor):
...
+        try:
+            pm_data = self.pms5003.read()
+        except pmsReadTimeoutError:
+            logging.warn("Failed to read PMS5003")
+        else:
+            pm1 = pm_data.pm_ug_per_m3(1.0)
+            pm2 = pm_data.pm_ug_per_m3(2.5)
+            pm10= pm_data.pm_ug_per_m3(10.0)
+
+        if "ENVIRO_PLUS_PM" in os.environ:
```
Comment (shaunmulligan) [inline sensor/scripts/enviroplushat.py]: Not a huge fan of having this stuff all repeated, but could think of a quick cleaner way yet

File: sensor/scripts/enviroplushat.py [hunk 2]
```diff
@@ -36,17 +44,46 @@ def get_readings(self, sensor):
...
+
+        try:
+            pm_data = self.pms5003.read()
+        except pmsReadTimeoutError:
+            logging.warn("Failed to read PMS5003")
+        else:
+            pm1 = pm_data.pm_ug_per_m3(1.0)
+            pm2 = pm_data.pm_ug_per_m3(2.5)
+            pm10= pm_data.pm_ug_per_m3(10.0)
+
```
Comment (rahul-thakoor) [inline sensor/scripts/enviroplushat.py]: shouldn't this also be executed only `if "ENVIRO_PLUS_PM" in os.environ`

**Comment by rahul-thakoor** [comment]
I couldn't get it to work on Pi4 with prod image and all settings configured.

**Comment by peteretep** [comment]
what is the status of this? Just got an enviro pi+ with PM sensor so would love to get it up and running with balena.

**Pull request closed without merging**